python/artm/regularizers.py

```diff
@@ -973,28 +974,27 @@ def __init__(self, name=None, window=None, threshold=None, background_topic_name
 
         BaseRegularizer.__init__(self,
                                  name=name,
-                                 tau=1.0,
+                                 tau=tau,
                                  gamma=None,
                                  config=config)
         if window is not None:
             self._config.window = window
             self._window = window
-        elif config is not None and config.HasField('window'):
-            self._window = config.window
 
         if threshold is not None:
             self._config.threshold = threshold
             self._threshold = threshold
-        elif config is not None and config.HasField('threshold'):
-            self._threshold = config.threshold
 
         if background_topic_names is not None:
             if isinstance(background_topic_names, string_types):
                 background_topic_names = [background_topic_names]
             for topic_name in background_topic_names:
                 self._config.background_topic_names.append(topic_name)
-        elif config is not None and len(config.background_topic_names):
```

Comment:
Why did you delete loading of parameters from config? Also you should add class fields for new parameters, add setters/getters  and make parameter loading from config for them too.